package/drop_versions.py

```diff
@@ -19,7 +19,9 @@ def drop_version(todrop, obj):
     out = [];
     for o in obj:
         version = o['version'].encode('ascii')
-        if version != todrop:
+        if version == todrop:
+            print("Dropping version {0}".format(todrop), file=sys.stderr)
```

Comment:
This, unfortunately, does not work in Python 2.7.  Is there some compatibie way for V2 and V3 to make this happen?  The rest of the scripts are python version agnostic, so I'd rather not make this a V3 only.

Reply:
Was the script version agnostic **before** application of #6070 ?

Reply:
You're right, I think I goofed.  It already had that type of print formatting.